### DISCUSSION TITLE
Multi-select when using dynamic tag for CSS Classes

### DISCUSSION THREAD

**Discussion by numediawebsites** [elementor/elementor#27850] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I can type multiple classes for CSS Classes under Advanced when editing an element, but when selecting a dynamic tag, I can only select one.

### Describe the solution you'd like.

I'd like to be able to select multiple dynamic tags for CSS Classes.

### Describe alternatives you've considered.

I could imaging that CSS Selectors in the Roadmap might address this.

My solution, when registering my own custom Dynamic Tag class, is to use a SELECT2 control, which works, however...

### Additional context

This is the use-case I'm working with:

In the list of classes made available by the custom Dynamic Tag class written, there are two styles for shadows, and two styles for borders. Ideally the user should only be able to select ONE shadow style, and/or ONE border style - in other words, each of shadow and border styles would separately registered custom dynamic tag classes. Rather than being able to select BOTH of each, which obviously doesn't make sense. 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by catchlightWeb** [comment]
I could be misunderstanding the request, but can't you put multiple classes in the class field. "class-one class-two"?

**Reply by numediawebsites** [reply]
You can manually type in multiple classes, but you cannot select multiple classes using the Dynamic Tag feature, unless I'm mistaken.

**Reply by catchlightWeb** [reply]
I believe you're correct. I didn't completely understand your request.
